src/Queue.php

```diff
@@ -225,6 +225,16 @@ public function getWorkerPid()
     protected function handleMessage($id, $message, $ttr, $attempt)
     {
         list($job, $error) = $this->unserializeMessage($message);
+
+        // Handle aborted jobs without thrown error
+        if ($attempt > 1) {
```

Comment:
I think that all this can be combined into one condition

Reply:
@s1lver Shall I summarize that? I'd be happy to. I personally prefer the breakdown for quick readability.

Reply:
@s1lver Thanks for the example.

I've summarized the condition here:
https://github.com/yiisoft/yii2-queue/pull/528/commits/99d9f892df7a340aaae5e61f6c23361c0eea8243

I'm not sure about the `return` in your example, since we only return `true` if the condition matches.